tests/spec/SpecHelper.js

```diff
@@ -171,9 +172,9 @@ function makeDefaultBody() {
   return "<section id='sotd'><p>foo</p></section><section id='toc'></section>";
 }
 
-function makeStandardOps() {
+function makeStandardOps(configParams) {
```

Comment:
So, we probably want to be able to override the body too... so: 

```JS 
function makeStandardOps(config = {}, body = makeDefaultBody()) {
   return { 
      body, 
      config: {...makeBasicConfig(), ...config}
   };
}
```

Reply:
the body part too needs to be appended right? or completely overridden ?

Reply:
overridden, as body is just a string.

Reply:
Fine :+1: